Automattic/wp-calypso
# Screen options tab component with tests and feature switch

#### Changes proposed in this Pull Request

* Create the Screen Options tab and the Screen Selector component in Calypso.

#### Testing instructions



* Visit Calypso.live link generated below and append `?flags=nav-unification/switcher` to the end of the URL to enable the Screen Options
* Test the component looks like the design and you are able to interact with it using mouse & keyboard as you'd expect.
* Test that it has been added to all the correct pages listed here https://github.com/Automattic/wp-calypso/issues/53706. With the exclusion of the list below.

![Screenshot 2021-06-21 at 10 56 23](https://user-images.githubusercontent.com/8639742/122743884-5b566400-d27f-11eb-91f3-3ec75fe5fae3.png)




#### Additional information

The following views from the related issue have not had the Screen Options component added to them:
* Posts > Add new
* Pages > Add new
* Appearance > Customizer
* Appearance > Widgets
* Appearance > Menus
* Appearance > Background
* Site editor (FSE sites)

Fixes https://github.com/Automattic/wp-calypso/issues/53706
